bitcoin-magazine/bitcoin-explained/accounts-for-bitcoin-easypaysy!.md

```diff
@@ -0,0 +1,1038 @@
+So it's compromising everyone's privacy.
+So while it's very convenient, it's something we shouldn't be doing, is it, Sjors?
+
+Sjors: 00:02:17
+
+Exactly.
+So that's why most wallets have a button called get new address and some good wallets will actually automatically make a new address whenever you even click on receive. Some wallets don't.
+All right
+
+Aaron: 00:02:30
+
+So that works we need We need to generate a new address for each payment we receive.
+However, Sjors, this is a little bit inconvenient as well, isn't it?
+
+Sjors: 00:02:38
+
+It can be.
+
+Aaron: 00:02:39
+
+So, here's the solution.
+We need an account system for
+
+## Why accounts would be useful intro to Easypaysy
+
+Bitcoin.
+Or at least, it's possible to create an account system on Bitcoin and it could be useful.
+
+Sjors: 00:02:49
+
+I think the the word account has some negative connotations, but it'd be nice if you knew one thing about the other party and then you can keep sending the money while automatically using a new address.
+I think that's what we want.
+
+Aaron: 00:03:05
+
+Right.
+So, Jose Femenas Canoelo, that's probably a severe mispronunciation of his name, but Spanish developer Jose Femenas Canoelo, he also believes so.
+He believes Bitcoin would benefit from an account system.
+It's called Easypaysy, which is short for Easy Payment System, I think.
+Easypaysy, And that's what we're going to discuss today.
+
+Sjors: 00:03:33
+
+All right.
+
+Aaron: 00:03:33
+
+Okay.
+So it's pretty clever and we'll get to possible downsides later, but let's first just explain how it works.
+
+Sjors: 00:03:41
+
+Yeah.
+Yeah.
+How does it work?
+Because this time you're going to explain how this thing works.
+
+Aaron: 00:03:44
+
+I'm going to do
+
+## what is Easypaysy?
+most of the explaining this time.
+All right, so to generate an account, one of the clever things about this proposal is that it uses nothing outside of the Bitcoin blockchain itself, necessarily.
+There's options to that, but we can do it all without any outside of blockchain thing.
+
+Sjors: 00:04:06
+
+Well, by thing you don't mean wallets, but you mean there's no communication protocol, you don't have to run a server to communicate with the other person?
+
+Aaron: 00:04:14
+
+Exactly, we generate the account on the Bitcoin blockchain itself, and people can find the account on the Bitcoin blockchain itself.
+
+Sjors: 00:04:22
+
+Okay, so how do I generate an account?
+
+Aaron: 00:04:24
+
+Right, so an account is basically a transaction.
+That's in this proposal what an account is.
+And this transaction consists of one input, so one address where coins are sent from, but this one input is a multi-sig input, so it has two public keys.
+It also has one output, which is a, what's it called?
+No, OP_RETURN, sorry.
```

Comment:
Good catch of the AI's transcription error. For code/bitcoin-related technical terms such as opcodes like `OP_RETURN`, we try to enclose them in backticks (`) for clarity.